actions.yaml

```diff
@@ -0,0 +1,7 @@
+cis-benchmark:
+  description: CIS Kubernetes Benchmark
+  params:
+    mode:
+      type: string
+      default: 'run'
+      description: Action mode. Defauls to 'run'.
```

Comment:
what are the options here? defaults to run, but possibles are ...

Reply:
@hyperbolic2346 just 'run' today.  We'll also have 'apply' soon(ish) that will do the `snap set` to fix reported failures, so I went ahead and put the param in to prepare for that.  I can the whole `mode` param out for now if you think it'll confuse people.

Reply:
I'm fine with leaving it in if the plan is to add to it and we just haven't done that yet. When we add to it, we should be sure to update the description.